gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Gherkin/CommonSteps.cs

```diff
@@ -112,13 +112,16 @@ public void TranslateTraversal(string traversalText)
             {
                 throw new InvalidOperationException("g should be a traversal source");
             }
-            _traversal = TraversalParser.GetTraversal(traversalText, _g, _parameters);
```

Comment:
Can't we now remove the `TraversalParser` logic completely?

Reply:
technically - yes. i just didn't feel right removing it on this PR for some reason. the code is all pretty useless now i think. i'll get rid of it.